dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -595,17 +612,12 @@ public void run() {
                     Logger.error(HostAPIImpl.class, e.getMessage(), e);
                     throw new DotRuntimeException(e.getMessage(), e);
                 }
+                finally {
+                    DbConnectionFactory.closeSilently();
```

Comment:
if you prefer, you can replace this by annotating the method run with ClosedDbIfNeeded annotation